src/vdaf.rs

```diff
@@ -248,6 +248,21 @@ pub trait Aggregator<const VERIFY_KEY_SIZE: usize, const NONCE_SIZE: usize>: Vda
         agg_param: &Self::AggregationParam,
         output_shares: M,
     ) -> Result<Self::AggregateShare, VdafError>;
+
+    #[cfg(feature = "experimental")]
+    /// This is special dpsa-project functionality.
```

Comment:
Answering in order:
 1. "dpsa" stands for "differentially private secure aggregation" and is just the acronym of our current project. This comment was not intended to be included in the PR and we will remove any mention to "dpsa".
 2. Adopting the terminology of the linked paper, our approach is *Server-DP*, as each of the aggregators adds noise. This `postprocess()` function is part of the `Aggregator` trait, and as such I would say that implementing it for a VDAF would only be useful in the context of Server-DP. 
 3. Yes, this seems useful. This has to be coordinated with a change in janus to supply this parameter, which shouldn't be too complicated.
 4. Yes, I like the name `apply_differential_privacy_noise()` as suggested by Tim below.

**NOTE**, remark for (2.):
Instead of this API on the `Aggregator` trait, is definitely worth to reconsider our API on the `Prio3` trait. We currently have an `add_noise()` function, which is just as useful in a Local-DP or Client-DP setting, as it is in our Server-DP setting. A small caveat to be noted is that, for local noising, there is theoretically also the option to add noise directly on the measurement type, which is not implementable using `add_noise()`.

Reading the paper, it would be quite easy to modify our API to accommodate for DPrio-style noising as well: We would replace `add_noise()` by
```rust
fn generate_noise(&self) -> Result<Vec<F>, FlpError>
```
then for Local-DP, Client-DP and Server-DP the noise would be added to the encoded input / share, and for DPrio the generated noise would be submitted to the aggregator seperately, as intended by that protocol.